tycho-client/src/feed/synchronizer.rs

```diff
@@ -62,8 +62,8 @@ pub struct ComponentWithState {
 
 #[derive(Clone, PartialEq, Debug, Default, Serialize)]
 pub struct Snapshot {
-    states: HashMap<String, ComponentWithState>,
-    vm_storage: HashMap<Bytes, ResponseAccount>,
+    pub states: HashMap<String, ComponentWithState>,
```

Comment:
this is not needed @tamaralipows! there are getter methods below!

Reply:
how did I miss this ehehehe sorry